bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/i18n/netatmo.properties

```diff
@@ -330,8 +330,7 @@ thing-type.netatmo.wind.description = Wind sensor reporting wind angle and stren
 
 conf-error-no-client-id = Cannot connect to Netatmo bridge as no client id is available in the configuration
 conf-error-no-client-secret = Cannot connect to Netatmo bridge as no client secret is available in the configuration
-conf-error-no-username = Cannot connect to Netatmo bridge as no username is available in the configuration
-conf-error-no-password = Cannot connect to Netatmo bridge as no password is available in the configuration
+conf-error-grant-needed = Configuration incomplete, please grant the binding to Netatmo Connect.
```

Comment:
Please run the i18n tool as there should be other' changes (parameter refreshToken, username, password).

Reply:
I did it.

Reply:
Ok, I think I understand, this is not something new in this PR but something we missed when reviewing your PR for the new binding.
As you have no static things declared (in XML) linked to these configs, the i18n tool does ignore them.
So you have no label/description for these configurations available in the translation file.
The solution would be, I believe, to use `@text` in the config/xml file and add the corresponding entries in the default translation properties.

Reply:
Ok, I think I understand, this is not something new in this PR but something we missed when reviewing your PR for the new binding.
As you have no static things declared (in XML) linked to these configs, the i18n tool does ignore them.
So you have no label/description for these configurations available in the translation file.
The solution would be, I believe, to use `@text` in the config/xml file and add the corresponding entries in the default translation properties.

Reply:
This can be fixed in a next PR if we want to merge the current PR very soon.

Reply:
...or by extending the translation tool to take this case in account ?

Reply:
The tool can do nothing as it will know what key to produce. That is typically the case where you need to use `@text/myparameterlabel` and in this case, the tool will add the entry `myparameterlabel` in the properties file.

Reply:
I am going to create a PR for that.